### ISSUE TITLE
Public Ticket Interface does not fetch and show ticket categories

### ISSUE THREAD

**Issue by joerglott** [Dolibarr/dolibarr#20160]
### Bug

Public Interface does not fetch and show ticket categories from the dictionary. But it is a MUST field. No ticket creation is possible in the public interface.

In the dictionary the categories are defined. I am using the ticketing since two years around and using the ticket categories. Now I want to use the public interface as well.

New in the 15 version (seems to be new - because I am not remembering this in the 14eth version)  is to create categories as well in the ticket area. This categories are displayed below in the public interface, but they are not that one, which are created in the dictionary. they are working - the question here is how useful is this for the business process? I can imagine that we do not need this at this point.

### Environment Version

15

### Environment OS

Ubuntu 20.04 LTS

### Environment Web server

Apache 2.4

### Environment PHP

PHP 7.4.21

### Environment Database

mysql-Server 8.2

### Environment URL(s)

https://knowhere.knowhere-consulting.eu/public/ticket/create_ticket.php

### Expected and actual behavior

Expected is that the created categories in the dictionary are being displayed in the drop down field to choose.

Current beahvier is that there is nothing in the drop down list. Only the text "No categories defined" (but we have some)

### Steps to reproduce the behavior

1. Open the public ticket interface
2. Go to create a new ticket
3. Go to Ticket Category an do open it - it is empty but filled in dictionary.

### Attached files

![dolibarr_ticket_error](https://user-images.githubusercontent.com/89402648/155304586-70ee73cd-2861-4d28-b641-7ea651d56d30.jpg)
![Screenshot von 2022-02-23 10 43 18](https://user-images.githubusercontent.com/89402648/155304601-141bbb14-2831-43ab-9781-bb353c00e998.jpeg)
![Screenshot von 2022-02-23 10 43 56](https://user-images.githubusercontent.com/89402648/155304610-bc3d3554-19bb-4ce0-b0e6-949f787bf20f.jpeg)

**Comment by atm-maxime** [comment]
Did you try to set "grüuppe ist öffentlich" to yes to have it available in the public interface?

**Comment by joerglott** [comment]
Hi Maxime,
did I missed a adjustment? Where do I need to adjust this?
Thanks
Joerg

**Comment by atm-maxime** [comment]
In the dictionary you have to define if the category is available on public interface

**Comment by joerglott** [comment]
F*ck. Sorry. It is so long ago that I defined them and did not recognized yesterday that there is an entry. I changed and it is working now. Thanks for your help and sorry for my stupid question. Yesterday I did not saw this in my check session. Thanks.

Last question: We have another Category definition here - in my point of view it is not needed for business processes here. Is there any other thought why using category here again?
